src/cli/MQBaseHandler.ts

```diff
@@ -89,13 +92,11 @@ export default abstract class MqBaseHandler implements ICommandHandler {
      *
      * @param {IHandlerParameters} commandParameters Command parameters sent to the handler.
      * @param {Session} session The session object generated from the mq profile.
-     * @param {IProfile} mqProfile The mq profile that was loaded for the command.
      *
      * @returns {Promise<IMQResponse>} The response from the underlying mq api call.
      */
     public abstract async processWithSession(
         commandParameters: IHandlerParameters,
-        session: Session,
-        mqProfile: IProfile
```

Comment:
I initially thought this could've been a breaking change but since we do not export anything on the `src/cli` folder, we are fine 😋 
Still good to do as part of vNext though since it's kind of a big change
(Same as IMS)